CHANGELOG.rst

```diff
@@ -1,9 +1,16 @@
 Changelog
 =========
 
-0.9.0 (unreleased)
+1.0.0 (unreleased)
```

Comment:
Yes but at somepoint we need to make the move. Today seems a good day to stabilize the kinto-attachment API.